server/cursor.js

```diff
@@ -0,0 +1,71 @@
+import {check, Match} from 'meteor/check';
+import {Meteor} from 'meteor/meteor';
+
+import {Cursor} from '/lib/cursor';
+import {User} from '/lib/user';
+import randomColor from 'randomcolor';
```

Comment:
Imports are structured in sections based on where they come from. `randomcolor` is a third-part library so it should come after Meteor but before our modules. See [code style](https://github.com/peer/doc/blob/master/CODE_STYLE.md#imports).